src/legacy/core_plugins/region_map/public/region_map_type.js

```diff
@@ -60,23 +62,23 @@ provided base maps, or add your own. Darker colors represent higher values.' }),
     requiresUpdateStatus: [Status.AGGS, Status.PARAMS, Status.RESIZE, Status.DATA, Status.UI_STATE],
     visualization: RegionMapsVisualization,
     editorConfig: {
-      optionsTemplate: '<region_map-vis-params></region_map-vis-params>',
+      optionsTemplate: (props) => <RegionMapOptions {...props} serviceSettings={serviceSettings} regionmapsConfig={regionmapsConfig} />,
       collections: {
         legendPositions: [{
```

Comment:
Currently the `legendPositions` as well as `addTooltip` params are not used in the `Region Map` vis editor.
Do we need to include it as in `Tile Map` options ? :
 
![image](https://user-images.githubusercontent.com/31325372/62781733-4aecd400-bac1-11e9-943c-036b611c1324.png)

Especially since we have a separate component for that purpose -> [`BasicOptions`](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/basic_options.tsx);
It will look like:

![legend_positions](https://user-images.githubusercontent.com/31325372/62781967-d5cdce80-bac1-11e9-9a34-9d70b6dd0d66.png)

@timroes @cchaos please, respond with your thoughts 🙏

Reply:
I'd say that if a setting doesn't actually do anything to the particular chart type then it will only confuse users and it should not be available.